testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -159,33 +187,35 @@ Feature: Smoke tests for <client>
     When I wait until "prometheus" service is active on "<client>"
     And I visit "Prometheus" endpoint of this "<client>"
 
-  Scenario: Test Prometheus exporter formula on the <client>
+@monitoring_server
+  Scenario: Test <client> as Monitoring Server using Grafana formula
```

Comment:
We need to remove completely this scenario as agreed this morning

Reply:
same as above, only the monitoring server